.gitignore

```diff
@@ -243,4 +243,4 @@ define_sanity_output.txt
 /config/ezdb.txt
 
 # Running OpenDream locally
-tgstation.json
+tgstation.json
```

Comment:
this change shouldn't be here would you mind discarding your changes to the gitignore file